content/en/docs/how-tos/adding-a-new-secret-to-ci.md

```diff
@@ -3,27 +3,33 @@ title: "Adding a New Secret to CI"
 description: How to self-service manage secret data provided to jobs during execution.
 ---
 
-Jobs execute as `Pod`s; those that need access to sensitive information will have access to it through mounted Kubernetes
+Jobs execute as `Pod`s; those jobs that need access to sensitive information can have access through mounted Kubernetes
 [`Secrets`](https://kubernetes.io/docs/concepts/configuration/secret/). Secret data is managed self-service by the owners
 of the data.
 
 ## Add A New Secret
 
-In order to add a new secret to our system, you will first need to create a secret collection. Secret collections are managed
-at [selfservice.vault.ci.openshift.org](https://selfservice.vault.ci.openshift.org). Just head there, log in, create a new
-one and ideally also add your teammates as members. Important: Secret collection names are globally unique in our system.
+In order to add a new secret to our system, you will first need to create a "secret collection". Secret collections are managed
+at [selfservice.vault.ci.openshift.org](https://selfservice.vault.ci.openshift.org). Just head there, log in, click on the
+"New Collection" button, enter a name for your new secret collection, and click the "Submit" button and, ideally, add your teammates as members.
+Important: Secret collection names are globally unique in our system.
 {{< alert title="Info" color="info" >}} Users must have logged in to the DPTP Vault system at least once before they are listed as potential members. {{< /alert >}}
 
 The secrets themselves are managed in our Vault instance at [vault.ci.openshift.org](https://vault.ci.openshift.org).
-You need to use the OIDC auth to log in there. After logging in, click on `kv`, then `selfservice` and you should see your secret collection.
+You need to use the OIDC auth to log in there (leave the Role as blank/Default). After logging in, click on `kv`, and you should see your secret collection.
```

Comment:
You still need to go to `kv/selfservice` to get to the collection, don't you?

Reply:
There is no "selfservice" to click even though I see "selfservice/technical-release-team/" in the secret path.

Reply:
Interesting, that may be due to permission differences.

In any case, if that's what the regular user sees, let the text reflect that.  Ignore this comment.